internal/api/grpc/management/idp_converter.go

```diff
@@ -518,3 +529,16 @@ func bindingToCommand(binding idp_pb.SAMLBinding) string {
 		return ""
 	}
 }
+
+func signatureAlgorithmToCommand(signatureAlgorithm idp_pb.SAMLSignatureAlgorithm) string {
+	switch signatureAlgorithm {
+	case idp_pb.SAMLSignatureAlgorithm_SAML_SIGNATURE_RSA_SHA1:
```

Comment:
If you handle the in the proto defined `SAMLSignatureAlgorithm_SAML_SIGNATURE_UNSPECIFIED` with `""` as an empty string, you can also remove the pointer in the command side, it will be removed in the marshalling of the event due to the `omitempty` json tag.